weblate/trans/tasks.py

```diff
@@ -153,22 +154,16 @@ def commit_pending(
     else:
         components = Component.objects.filter(translation__pk__in=pks)
 
-    # All components with pending units
+    hours_expr = F("commit_pending_age") if hours is None else Value(hours)
+    age_cutoff = ExpressionWrapper(
+        Now() - hours_expr * timedelta(hours=1), output_field=DateTimeField()
+    )
```

Comment:
This doesn't work for MySQL/MariaDB:

```
query = b'SELECT DISTINCT `trans_component`.`id`, `trans_component`.`name`, `trans_component`.`slug`, `trans_component`.`proje...nge`.`timestamp` < (CURRENT_TIMESTAMP(6) - (`trans_component`.`commit_pending_age` * INTERVAL 3600000000 MICROSECOND))'
...
django.db.utils.ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '))' at line 1")
```

Probably you can't multiply a field by an INTERVAL there.

Reply:
We deploy on PostgreSQL only, but last time this was brought up, several community members wanted to keep MySQL support. The previous discussion on this topic happened in https://github.com/WeblateOrg/weblate/issues/2825.

We don't have numbers for all installs because we do not track them for privacy reasons.

Reply:
Created https://github.com/WeblateOrg/weblate/issues/16140 for that.